### PR TITLE
[PREVIEW]RDM-3050 Use ContentUri to read content

### PR DESCRIPTION
**Pull request by adr1ancho**
### JIRA link (if applicable) ###

RDM-3050

### Change description ###

Use ContentUri to read content

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: application/src/main/java/uk/gov/hmcts/dm/service/BlobStorageReadService.java [hunk 1]
```diff
@@ -25,9 +27,8 @@ public BlobStorageReadService(CloudBlobContainer cloudBlobContainer) {
 
     public void loadBlob(DocumentContentVersion documentContentVersion, OutputStream outputStream) {
         try {
-            CloudBlockBlob blob = cloudBlobContainer.getBlockBlobReference(documentContentVersion.getId().toString());
-            blob.download(outputStream);
-        } catch (URISyntaxException | StorageException e) {
+            copy(new URL(documentContentVersion.getContentUri()).openStream(), outputStream);
```
Comment (rafalkalita) [inline application/src/main/java/uk/gov/hmcts/dm/service/BlobStorageReadService.java]: I disagree. Why to copy directly from URL? cloudBlobContainer is configured with connection string. And we use cloudBlobContainer everywhere. We should keep it for consistency and security.

Comment (adr1ancho) [reply application/src/main/java/uk/gov/hmcts/dm/service/BlobStorageReadService.java]: The old way relies on the logic getBlockBlobReference(documentContentVersion.getId()

It won't be consistent for my next change that breaks that rule

Comment (rafalkalita) [reply application/src/main/java/uk/gov/hmcts/dm/service/BlobStorageReadService.java]: But still this is secured with Azure connection string. I think it should stay like that. Please check with Valentin.

**Comment by adr1ancho** [comment]
Can't get Rafa to understand why we need the change.  Let's wait until things break apart and we can then apply the fix

**Pull request closed without merging**